src/App.scss

```diff
@@ -2,7 +2,125 @@ iframe {
   display: none;
 }
 
+.title {
+  font-size: 50px;
+}
+.containerForApp {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App{
+  border: 2px black solid;
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+  align-items: center;
+}
+
+.formUsers {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+
+  .field {
+    font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-size: 30px;
+      .form-field {
+        display: block;
+        width: 100%;
+        background-color: aqua;
+        padding: 8px 16px;
+        line-height: 25px;
+        font-size: 14px;
+        font-weight: 500;
+        font-family: inherit;
+        border-radius: 6px;
+        -webkit-appearance: none;
+        border: 1px solid var(--input-border);
+        transition: border .3s ease;
+        &::placeholder {
+            color: var(--input-placeholder);
+        }
+        &:focus {
+            outline: none;
+            border-color: var(--input-border-focus);
+        }
+    }
+    
+  }
+}
+
+.lableTitle {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.labelUser{
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.button-26 {
+  width: 200px;
+  appearance: button;
+  background-color: #1652F0;
+  border: 1px solid #1652F0;
+  border-radius: 4px;
+  box-sizing: border-box;
+  color: #FFFFFF;
+  cursor: pointer;
+  font-family: Graphik,-apple-system,system-ui,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
```

Comment:
nit: Don't use a specific font when it is inherited from the body, remove the `font-family` here.